### DISCUSSION TITLE
Licencing question

### DISCUSSION THREAD

**Discussion by bbassac** [d3/d3#3500] [Q&A] [answered]
Hi, Your code is using a lib called 'colorbrewer' wich is marked as GPL.
So , what is the impact about your licence (BSD-3-Clause License)? 

What are the limitations ?

**Comment by Fil** [comment, marked as answer]
Can you clarify what you are referring to?

As far as I know, D3 is using the colorbrewer color *schemes*, but does not include them as a "lib". At any rate, the color brewer website links to a repo that appears to be licenced under the APL, not GPL.

**Reply by bbassac** [reply]
Maybe I misunderstood something.
I have a file in lib/colorbrewer/LICENSE who talk about GPL-1.0-or-later
It's a warn from a Licence scanning tool.

**Reply by bbassac** [reply]
I think the scanner is broken, my bad.
It scans this : https://github.com/d3/d3/blob/v3.5.17/lib/colorbrewer/LICENSE
Or it's an apache, not a GPL .....
My BAD

**Reply by Fil** [reply]
No problem! Note that the current version of D3 is version @6. The relevant LICENCE/copyright information for Cynthia Brewer's colors is available in https://github.com/d3/d3-scale-chromatic/blob/master/LICENSE
